manifests/params.pp

```diff
@@ -39,4 +39,10 @@
   String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
   Boolean $pulp2_support = $katello::globals::pulp2_support,
 ) inherits katello::globals {
+
+  if !$pulp2_support {
```

Comment:
I avoid the pattern `if !$bool { first } else { second }` and always write `if $bool { second } else { first }`. However, in this case the whole if can be avoided. See the suggestion below.